stock_putaway_product/models/product.py

```diff
@@ -0,0 +1,45 @@
+# © 2016 Jos De Graeve - Apertoso N.V. <Jos.DeGraeve@apertoso.be>
+# © 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from lxml import etree
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    product_putaway_ids = fields.One2many(
+        comodel_name='stock.product.putaway.strategy',
+        inverse_name='product_tmpl_id',
+        string="Product stock locations")
```

Comment:
not blocking but I would be consistent with formatting and change the last row to 
```
string="Product stock locations",
)
```